src/roiextractors/example_datasets/toy_example.py

```diff
@@ -7,10 +14,39 @@
 
 
 def _gaussian(x, mu, sigma):
+    """Compute classical gaussian with parameters x, mu, sigma."""
     return 1 / np.sqrt(2 * np.pi * sigma) * np.exp(-((x - mu) ** 2) / sigma)
 
 
 def _generate_rois(num_units=10, size_x=100, size_y=100, roi_size=4, min_dist=5, mode="uniform"):
+    """Generate ROIs with given parameters.
+
+    Parameters
+    ----------
+    num_units: int
+        Number of ROIs
+    size_x: int
+        Size of x dimension (pixels)
+    size_y: int
+        Size of y dimension (pixels)
+    roi_size: int
+        Siz of ROI in x and y dimension (pixels)
+    min_dist: int
```

Comment:
I know @bendichter has a strong preference towards the `numpydoc` style guide for describing default values (and/or optionality) like so https://github.com/catalystneuro/neuroconv/pull/260#discussion_r1055557321

Reference: https://numpydoc.readthedocs.io/en/latest/format.html#parameters

I'll grant you, the ongoing PEP 727 discussions (courtesy @h-mayorquin for point it out) emphasize these are pseudo-standards at best, which makes linting/parsing very difficult: https://discuss.python.org/t/pep-727-documentation-metadata-in-typing/32566/14

So similarly to my other comment, I'm fine with this being a 'first step' but we should reach agreement beyond pydocstyle about these other things too

Reply:
I think Ben's threshold, for which another set of Python tests could be added similar to your inheritance tests, is whether or not the numpy docstring parser library can successfully scrape the docstring to essentially reproduce the method signature

Reply:
Though I see for the other docstrings you did get into the habit of writing it in that way - maybe it's just these earlier once that didn't follow that convention?

Reply:
Though a part of Ben's opinion on this kind of thing also comes down to the spacing of the colon, such as

```python
     roi_size : int, optional
           # ^ ^
        Size of ROI in x and y dimension (pixels).
```

Reply:
As per conversation: will look into other linters that are more specific than `pydocstyle`